core/src/voting_service.rs

```diff
@@ -23,6 +23,8 @@ use {
     thiserror::Error,
 };
 
+// Convservatively allow 50 TPS per validator.
```

Comment:
Might be good to spell out the "why" a little clearer.

For example:
```
// With 400ms slot times, a validator typically only needs to vote 2.5 times per second,
// but we overprovision this rate limit by an order of magnitude so that we don't accidentally
// throttle votes, which could be catastrophic. 50 is also sufficiently low to prevent some
// malicious validator DoS attack from impacting some victim validator.
```